docs/blog/Webots-2023-b-release.md

```diff
@@ -0,0 +1,112 @@
+
+All options and instructions on how to use this new launcher are available in the [User Guid](../guide/running-extern-robot-controllers.md).
+
+---
+
+## Overhaul of the PROTOs Documentation
+
+All the documentation concerning robots, sensors, actuators and objects have been moved to [webots.cloud](https://webots.cloud/proto).
+
+Not only is it now easier to navigate through the PROTOs, but you can also change and test the parameters of PROTOs directly online.
+
+You can also add your own PROTO model to the library such that everybody can use it.
+
+Read more about the new documentation system for PROTOs [here](https://cyberbotics.com/doc/guide/webots-cloud?#share-and-use-your-proto).
+
+---
+
+## ROS2
+
+With this release we continue to develop and improve the support for ROS 2 in Webots.
+New features include:
+
+- Added support for ROS 2 Iron Irwini.
+- Added interface for new devices: VacuumGripper, Pen, Compass, Emitter and Receiver.
+- Added a new node to start nodes only when Webots is ready.
+- Added a new node to start robot plugins in a cleaner way.
+- Added a new TIAGo launch file for full configuration.
+
+And many bug fixes.
+You can see all the changes in the [main changelog](https://github.com/cyberbotics/webots_ros2/blob/master/webots_ros2/CHANGELOG.rst) of the package.
+
+---
+
+## Extra Goodies
+
+- A [new script](https://github.com/cyberbotics/webots/tree/master/scripts/proto_formatter), integrable in VS Code, to correct the indentation of your PROTO files.
+- Robot's gripper now used coupled motors.
+
+**Go and [download Webots R2023b](https://cyberbotics.com/#download) today, so you do not miss out on all these great new features!**
+
+---
+
+## Acknowledgements
+
+The current release includes contributions from [Daniel Dias](https://github.com/ad-daniel), [Kimberly McGuire](https://github.com/knmcguire), [Chandan Bharadwaj](https://github.com/Chandan-Bharadwaj), [Songyang Zhao](https://github.com/songyangZhao), [ShuffleWire](https://github.com/ShuffleWire), [Jakub Delicat](https://github.com/delihus), [Léo Duggan](https://github.com/Jean-Eudes-le-retour), [nilsjor](https://github.com/nilsjor), [Simon Gene Gottlieb](https://github.com/SGSSGene), [Tsubasaya](https://github.com/Minimerl), [Dean Brettle](https://github.com/brettle), [Chirath Pansilu](https://github.com/ChirathPansilu), [fparat](https://github.com/fparat), [Gaël Écorchard](https://github.com/galou), [Ian Burwell](https://github.com/IanBurwell), [Stephan Kunz](https://github.com/stepkun), [Kode Creer](https://github.com/kodecreer), [Toshiharu Tabuchi](https://github.com/toshiharutf), [Justin Beri](https://github.com/justinberi), [DrakerDG](https://github.com/DrakerDG), [Jolon Behrent](https://github.com/JolonB), [Darko Lukić](https://github.com/lukicdarkoo), [Angel Ayala](https://github.com/angel-ayala), [Sebastian Ohl](https://github.com/sebastianohl) and [GnSight](https://github.com/ftyghome).
```

Comment:
Is this list a copy paste from the previous blog or is updated to the people that actually contributed to R2023b?

Because I don't see some of the user that added some contributions in the last months and others that are listed didn't contribute explicitly to R2023b.